db/migrate/20200422180727_replace_contact_type_with_contact_types_on_case_contact.rb

```diff
@@ -0,0 +1,12 @@
+class ReplaceContactTypeWithContactTypesOnCaseContact < ActiveRecord::Migration[6.0]
+  def change
+
+    # NOTE: This is a destructive migration that we would normally avoid
+    #       if we were working on production data, but because there is
+    #       no production data we are comfortable being destructive and
+    #       losting whatever data is in the `contact_type` column.
+    remove_column :case_contacts, :contact_type
+    add_column :case_contacts, :contact_types, :string, array: true
+    add_index :case_contacts, :contact_types, using: :gin
```

Comment:
aha https://hashrocket.com/blog/posts/exploring-postgres-gin-index

Reply:
It isn't really valid to have no contact type on a contact- is there a way to add that requirement at a db level?

Reply:
We definitely could add a not null constraint; but not this week. Mind making an issue and tagging us in it?